vhost-device-console/src/main.rs

```diff
@@ -84,13 +90,15 @@ impl TryFrom<ConsoleArgs> for VuConsoleConfig {
             backend,
             tcp_port,
             socket_count,
+            max_queue_size,
         } = args;
 
```

Comment:
Shouldn't we validate `max_queue_size` here? The README states it must be greater than 0 and a power of 2 but there's no validation performed I think?

Reply:
We discussed here: https://github.com/rust-vmm/vhost-device/pull/811#discussion_r1952604259

I'm fine to re-introduce the check, I thought it was superfluous.